Week1/homework/showCurrentTime/showCurrentTime.js

```diff
@@ -0,0 +1,16 @@
+function showCurrentTime() {
+    var today = new Date();
+    var h = today.getHours();
+    var m = today.getMinutes();
+    var s = today.getSeconds();
+    m = checkTime(m);
+    s = checkTime(s);
+    document.getElementById('txt').innerHTML =
+    h + ":" + m + ":" + s;
+    var t = setInterval(showCurrentTime, 500);
+  }
+  function checkTime(i) {
+    if (i < 10) {i = "0" + i};  // add zero in front of numbers < 10
```

Comment:
Be careful of the indentation. 
```suggestion
    if (i < 10) {
       i = "0" + i
    }  // add zero in front of numbers < 10
```